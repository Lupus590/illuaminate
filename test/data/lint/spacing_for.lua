-- config: (lint (only (format:op-space format:separator-space)))

for i=0, 10 do end
for i= 0, 10 do end
for i = 0, 10 do end
for i = 0, 10 do end -- ok
