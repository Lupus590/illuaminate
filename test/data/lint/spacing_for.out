spacing_for.lua:[3:6-3:7]: Expected whitespace around "=". [format:separator-space]
   │
 3 │ for i=0, 10 do end
   │      ^
spacing_for.lua:[4:6-4:7]: Expected whitespace around "=". [format:separator-space]
   │
 4 │ for i= 0, 10 do end
   │      ^
No errors and 2 warnings
@@ -1,7 +1,7 @@
 -- config: (lint (only (format:op-space format:separator-space)))
 
-for i=0, 10 do end
-for i= 0, 10 do end
+for i = 0, 10 do end
+for i = 0, 10 do end
 for i = 0, 10 do end
 for i = 0, 10 do end -- ok
 
